stateproofs/stateProofVerifier_test.go

```diff
@@ -0,0 +1,46 @@
+package stateproofverification
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+	"github.com/algorand/go-algorand-sdk/encoding/json"
+	"github.com/algorand/go-algorand-sdk/types"
+)
+
+func readJsonFile(filePath string, target interface{}, assertions *require.Assertions) {
+	contents, err := ioutil.ReadFile(filePath)
+	assertions.NoError(err)
+
+	err = json.Decode(contents, &target)
+	assertions.NoError(err)
+}
+
+func TestStateProofVerification(t *testing.T) {
+	a := require.New(t)
+
+	prevStateProofFileName := "prevStateProof.json"
+	newStateProofFileName := "newStateProof.json"
```

Comment:
You can use `embed` to load these files

Reply:
Perhaps I'm missing something, but using embed causes CI to fail for go 1.15. It seems I can't do that.

Reply:
I don't think you're missing anything, let's remove 1.15 from CI.

Reply:
Done.